css/reveal.scss

```diff
@@ -1073,7 +1073,7 @@ $controlsArrowAngleActive: 36deg;
 	background: black;
 	visibility: hidden;
 	opacity: 0;
-	z-index: 10001;
+	z-index: 99;
```

Comment:
You changed the index of the pause overlay, but that should remain at 100.

The `.r-overlay` should move to 99 so that it appears below the pause screen.

Reply:
sorry for that, pls review once more.